### PR TITLE
:sparkles: Use of models and refactoring

### PR DESCRIPTION
**Pull request by sultanofwolves**

### CODE REVIEW

File: models/sport.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from mongoengine import Document
+from mongoengine import StringField
+
+
+class Sport(Document):
+    name = StringField(max_length=60, required=True)
+    resourceID = StringField(required=True)
```
Comment (JaysusM) [inline models/sport.py]: There's another field in Sport: icon.

File: resources/event.py [hunk 2]
```diff
@@ -30,23 +32,16 @@ def get(self):
...
-        return eval(dumps(event)), 200
+        event_data = request.get_json(force=True, silent=True)
+
+        new_event = Event_model(
+            creation_date=datetime.datetime.now(),
+            event_date=event_data['eventDate'],
+            title=event_data['title'],
+            description=event_data['description'],
+            courtID=event_data['courtID'],
+            creatorUID=event_data['creator']
```
Comment (JaysusM) [inline resources/event.py]: this was changed in "events" branch. Did you merge it first?

Comment (sultanofwolves) [reply resources/event.py]: can't see any "events" branch

File: models/sport.py (hunk 1 above)
Comment (sultanofwolves) [reply models/sport.py]: Fixed

**Pull request merged**